.github/workflows/deploy.yml

```diff
@@ -0,0 +1,71 @@
+name: Deploy
```

Comment:
I don't think this is a good name

Reply:
Do you have one you prefer? “Publish” perhaps?